sql/moz-fx-data-shared-prod/firefox_accounts_derived/docker_fxa_customs_sanitized_v1/metadata.yaml

```diff
@@ -1,5 +1,13 @@
 friendly_name: Docker Fxa Customs Sanitized
 description: |-
+  This table contains FxA customs (sanitized) events up until the migration from AWS to GCP was concluded.
+  There is a time window between 2023-09-07 and 2023-09-15 when we were receiving events from both deployments.
+  As the migration as concluded, new data will land and be available via the v2 of this table.
+
+  ---
+
+  Original description
+
   Sanitized version of docker_fxa_customs from FxA.
   PII is hashed.
```

Comment:
I'd suggest keeping this concise summary first, and putting the additional details as a secondary paragraph.